projects/storefrontstyles/scss/components/myaccount/_new-consent-management.scss

```diff
@@ -5,17 +5,17 @@
   }
 
   .header {
-    font-weight: $line-height-base * 375;
+    font-weight: var(--cx-font-weight-semi);
     font-size: $font-size-base * 1.375;
-    font-family: 'Open Sans';
-    margin-bottom: $font-size-base * 1.875;
+    font-family: $font-family-sans-serif;
```

Comment:
* Please use our `—cx-color` variables for colours and `@include type()` mixin for predefined fonts. We are no longer using Bootstrap variables.
* Please make sure all your CSS code follows the guidelines provided on this wiki page https://wiki.one.int.sap/wiki/display/spar/UI+Styling+and+Accessibility+Onboarding 
* Please reach out to Spartacus Theme Owner if any doubts and to follow up with code review: @morganm58